### ISSUE TITLE
Google Books back-up

### ISSUE THREAD

**Issue by huffpuffblow** [rmcrackan/Libation#1094]
How easy would it be to adapt your code to apply to other platforms, specifically in my case Google Books? I avoided the subscription set-up of Audible for a long time, and bought a lot of children's audiobooks for my kids on Google books. Now that they have their own offline player, I can't find a way for them to listen to their books from google play. Is this something that could be easily adapted to in your code?

tnx

**Comment by wtanksleyjr** [comment]
You should be able to download your audiobooks directly and unprotected from Google Play - go to the website for Google Play, click on the Books tab at the top, then click on "Your Library". Find the book you want, click the sideways "..." on it, and click "Export". Choose "High Quality" and you should get an m4a file downloaded.

**Comment by huffpuffblow** [comment]
This doesn't work for all books. Out of all the books I've bought I can
only use this method for 3 of them.

**Comment by rmcrackan** [comment]
tl;dr: sorry, but no

Longer reason: breaking encryption is a ton of work when it can be done at all. With audible owning something like 2/3 of the entire audiobook market, even the remaining major competitors are fighting for the scraps of a distant 2nd place. I'm not sure it's within my skill at all. And if it is, it would be more time and effort than I can spare for comparatively small payback. That said, I'm all in favor of this happening, and I do hope someone else will do it one day.
